rebar.config

```diff
@@ -3,6 +3,7 @@
 {erl_opts, [ debug_info
            , warnings_as_errors
            , warn_export_all
+           , {platform_define, "^[0-9]+", deprecated_types}
```

Comment:
platform_define  = OtpRelease ++ “-“ ++ SysArch ++ “-“ ++ Words.

where

OtpRelease = erlang:system_info(otp_release).
SysArch = erlang:system_info(system_architecture).
Words = integer_to_list(8 * erlang:system_info({wordsize, external})).


Erlang 16 and older starting from the letter but newer version 17,18,19 starting from the number